### PR TITLE
Cellpose fix metayml

### LINKED ISSUE #3237: [FIX] Cellpose meta.yml and optional pretrained model input
### Is your feature request related to a problem? Please describe

1) The Cellpose module currently has wrong meta.yml specifications for input files. This was overlooked during the final review.

2) If a user wants to specify a pre-trained model that he created.

### Describe the solution you'd like

1)  We simply need to remove `model` from the meta.yml.

2) A pre-trained model should be provided as an optional input parameter.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by FloWuenne**
## PR checklist

This PR fixes the problems mentioned in the issue:
- fixed the meta.yml statements
- added optional input file for user specified model

Closes #3237 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by FloWuenne** [comment]
@nf-core-bot fix linting pretty please :pray:

**Review by ramprasadn — APPROVED** [review]
LGTM! 🚀
